src/file-events/cpp/apple_fsnotifier.cpp

```diff
@@ -179,7 +179,6 @@ void Server::handleEvent(JNIEnv* env, char* path, FSEventStreamEventFlags flags)
         type = FILE_EVENT_UNKNOWN;
     }
 
-    logToJava(FINE, "Changed: %s %d", path, type);
```

Comment:
So all the logging now needs to happen in the callback?

Reply:
We log the native event coming in just a few lines above. I wanted to kill one of them, because we had two log messages for every event.